src/core/server/saved_objects/saved_objects_config.ts

```diff
@@ -13,12 +13,14 @@ export type SavedObjectsMigrationConfigType = TypeOf<typeof savedObjectsMigratio
 export const savedObjectsMigrationConfig = {
   path: 'migrations',
   schema: schema.object({
-    batchSize: schema.number({ defaultValue: 100 }),
+    batchSize: schema.number({ defaultValue: 1000 }),
```

Comment:
I've also increased the default v1 migrations batchSize, in all cases using 1000 document batch has resulted in faster migrations without any adverse impact. So having this number higher makes it a better fallback for incase v2 migrations don't work.

Reply:
Have we tested how much memory impact it has?

Reply:
We discussed this offline, but I only tested the Elasticsearch heap with `watch curl -s -X GET "elastic:changeme@localhost:9200/_cat/nodes?h=heap*\&v" -H 'Content-Type: application/json'.` and didn't see a difference between 100 / 1000. Kibana isn't responding to any requests at this point, so memory should be stable, but I'm not sure how close we might be to exceeding the default heap.